scm/build/build.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/sh -ex
 export GOOS=linux
-cd $GOPATH/src/github.com/nokia/danm
-glide install --strip-vendor
-go get -d github.com/vishvananda/netlink
-go get github.com/containernetworking/plugins/pkg/ns
-go get github.com/golang/groupcache/lru
+# Force turn on go modules feature becase the project is inside GOPATH
+export GO111MODULE=on
```

Comment:
with 1.13 this is the default right, so won't be needed

Reply:
Yes, you are right, the 1.13 Release notes states the same